### DISCUSSION TITLE
Vercel builds queued but never deploying

### DISCUSSION THREAD

**Discussion by tejlabseven** [vercel/community#4426] [Help] [answered]
### Summary

Not sure if I did something wrong but everything was working a few days ago and now when I push new commits to GitHub I can see the changes there and I can see in Vercel there is a new commit but it queues the build and never deploys. How do I get Vercel to actually build the site? 

Builds used to take seconds and have now been queued for over 25 mins with no deploy. 

I have tried cancelling and pushing other builds. I've made changes to other projects and tried them but same issue persists

### Steps to Reproduce

push an updated build to github, go to vercel, see new build started but stuck in queued status

**Comment by swarnava** [comment, marked as answer]
Hi,

This should be fixed now. Let us know if you are still seeing this problem.
